includes/admin/payments/orders.php

```diff
@@ -761,18 +851,26 @@ function edd_order_details_extras( $order = false ) {
 					</div>
 				<?php endif; ?>
 
-				<div class="edd-order-payment-key edd-admin-box-inside">
-					<label for="edd_payment_key" class="label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
-					<input type="text" name="payment_key" id="edd_payment_key" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+				<div class="edd-admin-box-inside">
+					<div class="edd-form-group">
+						<label for="edd_payment_key" class="edd-form-group__label label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
+						<div class="edd-form-group__control">
+							<input type="text" name="payment_key" id="edd_payment_key" class="edd-form-group__input" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+						</div>
+					</div>
 				</div>
 
 				<div class="edd-order-ip edd-admin-box-inside">
-					<label for="edd_ip" class="label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
-					<?php if ( edd_is_add_order_page() ) : ?>
-						<input type="text" name="ip" id="edd_ip" value="<?php echo esc_attr( edd_get_ip() ); ?>" />
-					<?php else : ?>
-						<span><?php echo edd_payment_get_ip_address_url( $order->id ); // WPCS: XSS ok. ?></span>
-					<?php endif; ?>
+					<div class="edd-form-group">
+						<label for="edd_ip" class="edd-form-group__label label"><?php esc_html_e( 'IP', 'easy-digital-downloads' ); ?></label>
```

Comment:
`label` class can also be removed here (there are more but I'll stop repeating myself).